climada/entity/exposures/nightlight.py

```diff
@@ -223,12 +224,11 @@ def download_nl_files(req_files=np.ones(len(BM_FILENAMES),),
             'connection and whether filenames are still valid.') from exc
     return dwnl_path
 
-def load_nightlight_nasa(bounds, req_files, year):
```

Comment:
This I noticed after running static code. This kwargs was not being used, even before the changes to this function.

Reply:
If you remove positional arguments, you break backwards compatibility for users. I'm not 100% familiar with this module, but I noticed that this change will even break code within CLIMADA:
https://github.com/CLIMADA-project/climada_python/blob/40a6b86bde05bcdbeddb908525deee07e9516f9e/climada/entity/exposures/black_marble.py#L296-L297

Reply:
I see, but then it means we should drop that keyword from this function too... that kwards is not used within the function...

Reply:
I can also leave it if it requires too many changes, but should be checked in the future I guess

Reply:
I can't judge because I'm not familiar with this module. If you're not sure whether code might break due to this change, you shouldn't change it or ask somebody who is familiar with it (e.g. the original authors). If you aren't able to solve this within this PR, you could leave a note `TODO: argument req_files is not used in this function`.

Reply:
I brought it back